_includes/google-analytics.html

```diff
@@ -0,0 +1,11 @@
+<script>
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
```

Comment:
because (a) just porting the basic blog for now, and it already had Goggle Analytics setup
(b) reports of o-tracking being an undocumented beast, so leaving it for (maybe) later